hoot-core/src/main/cpp/hoot/core/util/Log.cpp

```diff
@@ -83,12 +89,57 @@ static void cplErrorHandler(CPLErr eErrClass, int err_no, const char *msg)
 }
 
 Log::Log() :
-_level(Log::Info)
+_level(Log::Info),
+_decorateLogs(false)
 {
   qInstallMessageHandler(myLoggerFunction);
   CPLSetErrorHandler(cplErrorHandler);
 }
 
+void Log::init() const
```

Comment:
No idea why a bunch of the `Log::` functions were split between `Log.cpp` and the now deleted `LogGeneric.cpp` but I've combined them into one file now.